apps/api/src/applicant/external-api.service.ts

```diff
@@ -651,13 +650,28 @@ export class ExternalAPIService {
     }
   }
 
-  async getApplicants(start?: string, end?: string) {
+  async getApplicants(filter: IENUserFilterAPIDTO) {
+    const { from, to, limit, skip } = filter;
+    let queryString = `SELECT DISTINCT applicant_id  
```

Comment:
Why are we building a string like this rather than using queryBuilder? I'm fairly certain this could open the query up to SQL injections

Reply:
We do have some protections in place to protect against that (validator and the auth guard), but I am happy to rewrite this to use the query builder to be safe.